### PR TITLE
Initial implementation

### LINKED ISSUE #1722: Implement volume ls and rm tests

### PR DESCRIPTION
**Pull request by mhagen-vmware**
Volume rm is currently broken, but this is a first pass at initial implementation of the remaining volume tests.

Fixes #1722

### CODE REVIEW

**Comment by andrewtchin** [comment]
lgtm

File: tests/test-cases/Group1-Docker-Commands/1-21-Docker-Volume-LS.md [hunk 1]
```diff
@@ -12,23 +12,24 @@ This test requires that a vSphere server is running and available
...
-5. Issue docker volume ls -f dangling=true to the VIC appliance
-6. Issue docker volume ls -f name=test to the VIC appliance
-7. Issue docker create -v test:/test busybox to the VIC appliance
-8. Issue docker volume ls -f dangling=true to the VIC appliance
+2. Issue docker volume create --name=test to the VIC appliance
+3. Issue docker volume create --name=test2 to the VIC appliance
+4. Issue docker create -v test:/test busybox to the VIC appliance
+5. Issue docker volume ls to the VIC appliance
+6. Issue docker volume ls -q to the VIC appliance
+7. Issue docker volume ls -f dangling=true to the VIC appliance
```
Comment (matthewavery) [inline tests/test-cases/Group1-Docker-Commands/1-21-Docker-Volume-LS.md]: I will need to check these a little more. I have not added code for `-f` is not actually implemented yet so the filter will not be applied. Output will still come out as if a `docker volume ls` were issued. `image ls` works the same way at the moment.

Comment (mhagen-vmware) [inline tests/test-cases/Group1-Docker-Commands/1-21-Docker-Volume-LS.md]: Yep, that is why the implementation is commented out.

**Comment by matthewavery** [comment]
lgtm

**Comment by mhagen-vmware** [comment]
.

**Pull request merged**